_projects/fun-riot-features.md

```diff
@@ -0,0 +1,33 @@
+---
+name: "Fun features for Riot-Web, Riot-iOS and Riot-Android"
+desc: "Add new features to Riot-Web, Riot-iOS and Riot-Android"
+difficulty: "Medium"
+issues:
+mentors:
+ - dbkr
+ - giomfo
+ - manuroe
+initiatives:
+ - GSoC
+tags:
+ - New Features
+ - Riot
+ - Client
+---
+
+Riot is a flagship Matrix client; an Apache-licensed set of communication apps for Web (React), iOS & Android. Some nice refinements that would be good GSoC projects include:
+
+* Ability to share maps of locations! We have a PR from the community for Riot/Web (https://github.com/matrix-org/matrix-react-sdk/pull/596) - adding to mobile would be cool too!
+* Adding custom emojis to the message composer and timeline on all platforms!
+* Support for using multiple accounts simultaneously.
+* Quick actions on iOS for replying to notifications.
+* Rich-text Editor! Riot/Web has a Rich-text Editor thanks to GSoC 2016 - why not add to the mobile platforms?
+* Sending and displaying mathematics
```

Comment:
This one should have it's own project idea. Definitely not biased!!

Reply:
Maybe as a next strep we can filter all these into separate pages